packages/kbn-shared-ux-components/src/page_template/no_data_page/no_data_page_body/__snapshots__/no_data_page_body.test.tsx.snap

```diff
@@ -0,0 +1,55 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`NoDataPageBody render 1`] = `
+<Fragment>
+  <EuiText
+    textAlign="center"
+  >
+    <KibanaSolutionAvatar
+      iconType="elastic"
+      name="Elastic"
+      size="xxl"
+    />
+    <EuiSpacer
+      size="l"
+    />
+    <h1 />
```

Comment:
Hm, looks like we should either be requiring a `pageTitle` in NoDataPageBody or ensuring an empty `h1` is not rendered if there is no `pageTitle`.